Lib/test/test_tools/test_i18n/test_i18n.py

```diff
@@ -310,6 +337,24 @@ def test_calls_in_fstring_with_partially_wrong_expression(self):
         self.assertNotIn('foo', msgids)
         self.assertIn('bar', msgids)
 
+    def test_pygettext_output(self):
+        """Test that the pygettext output exactly matches a file."""
+        data_dir = Path(__file__).parent / 'data'
+        filenames = (('messages.py', 'messages.pot'),
+                     ('docstrings.py', 'docstrings.pot'),
+                     ('fileloc.py', 'fileloc.pot'))
+
+        for input_file, output_file in filenames:
+            with self.subTest(f'Input file: data/{input_file}'):
+                contents = (data_dir / input_file).read_text(encoding='utf-8')
+                with temp_cwd(None):
+                    Path(input_file).write_text(contents)
+                    assert_python_ok(self.script, '-D', input_file)
```

Comment:
I think we should use the long argument to avoid needing to re-check what ``-D`` stands for:
```suggestion
                    assert_python_ok(self.script, '--docstrings', input_file)
```